coveo-stew/coveo_stew/pydev.py

```diff
@@ -1,99 +1,88 @@
 """
-A PyDev is the "one ring to rule them all"; it's represented by a `pyproject.toml` file that links most/all of the
-python projects in a repository.
+Projects marked as pydev behave as "one ring to rule them all"; its `pyproject.toml` file links most/all of the
```

Comment:
Ça m'avait pris vraiment beaucoup de temps (et même un peu de recherche) pour comprendre ça quand j'ai lu le README de `coveo-stew`. Ici je trouve ça super clair. Donc, je suggère que tu update le  `coveo-stew/README.md` pour remplacer

> A repo-wide pyproject.toml is available at the root, which refers to the projects by path. This is a development bootstrap for development convenience, so that one can work in any of the projects of the repository without having to configure multiple environments in the IDE.

par

> Projects marked as pydev behave as "one ring to rule them all"; its `pyproject.toml` file links most/all of the python projects in a repository for developer convenience (so that one can work in any of the projects of the repository without having to configure multiple environments in the IDE).